docs/submission/submitting-molecular-data.md

```diff
@@ -60,17 +74,27 @@ storage.url=https://score.qa.argo.cancercollaboratory.org
 
 ### Understanding the Song metadata fields
 
-Song accepts metadata in JSON format (also referred as a `Song payload`), which is validated against standard JSON Schema to ensure data quality. The first step of submitting sequencing data to ARGO is to prepare the Song metadata payloads conforming to the most recent JSON schema that has been defined by the DCC. The latest `sequencing_experiment` JSON Schema can be found in the ARGO [github repository](https://github.com/icgc-argo/argo-metadata-schemas/blob/master/schemas/sequencing_experiment.json).
+Song accepts metadata in JSON format (also referred as a `Song payload`), which is validated against standard JSON Schema to ensure data quality. The first step of submitting sequencing data is to prepare the Song metadata payloads conforming to the most recent JSON schema that has been defined by the DCC. The latest `sequencing_experiment` JSON Schema can be found in the ARGO [github repository](https://github.com/icgc-argo/argo-metadata-schemas/blob/master/schemas/sequencing_experiment.json).

```

Comment:
@rosibaj did you want to remove all instances of the word "payloads"?

Reply:
no i need to disucss this with you and christina.  i thought about it a lot since our last meeting. i dont think it makes sense to to that quite yet without a more elegant way of defining the submiitted data.  maybe we can define pauyload somehow.

Reply:
ok...maybe make sure where you removed "payloads" before makes sense?